src/main/java/appeng/helpers/PatternHelper.java

```diff
@@ -72,6 +75,17 @@ public PatternHelper( final ItemStack is, final World w )
 		final NBTTagList inTag = encodedValue.getTagList( "in", 10 );
 		final NBTTagList outTag = encodedValue.getTagList( "out", 10 );
 		this.isCrafting = encodedValue.getBoolean( "crafting" );
+
+		// TODO: Remove backward compatibility handling with rv4 stable.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)